### PR TITLE
Update openscap to 0.4.7

### PR DESCRIPTION
**Pull request by ares**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.14
* [ ] 1.13
* [ ] 1.12

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ares** [comment]
Should be done, let's see how the build proceeds

File: rubygem-openscap/rubygem-openscap.spec [hunk 1]
```diff
@@ -12,7 +12,7 @@ Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: ruby(rubygems)
 Requires: rubygem(ffi) >= 1.0.9
 # require libopenscap.so.8 in an arch neutral way
-Requires: openscap >= 1.2.1
+Requires: openscap >= 1.2.9
```
Comment (domcleal) [inline rubygem-openscap/rubygem-openscap.spec]: Please also update the BuildRequires.

File: rubygem-openscap/rubygem-openscap.spec [hunk 2]
```diff
@@ -77,17 +77,6 @@ gem build %{gem_name}.gemspec
 gem install --local --install-dir .%{gem_dir} \
             --force %{SOURCE0} --no-rdoc --no-ri
 
-%check
-set +e
-rake test 2>&1 > test.output
```
Comment (domcleal) [inline rubygem-openscap/rubygem-openscap.spec]: Don't remove the tests - just keep the `rake test` and drop all of the test.output parsing (i.e. revert most of b1c9a79).

File: rubygem-openscap/rubygem-openscap.spec (hunk 1 above)
Comment (ares) [inline rubygem-openscap/rubygem-openscap.spec]: sorry I missed that, fixed too now

**Comment by domcleal** [comment]
Merged and building, thanks @ares. Will retag to 1.14.

**Pull request merged**